Dockerfile

```diff
@@ -139,7 +139,7 @@ ARG FIN_TOKEN=
 RUN git clone https://$FIN_TOKEN:x-oauth-basic@github.com/ROCmSoftwarePlatform/Fin.git $FIN_DIR
 WORKDIR $FIN_DIR
 # Can be a branch or a SHA
-ARG FIN_BRANCH=260d9465d5ee71cc8dc7715ba583107ac4556f93
+ARG FIN_BRANCH=04508bcfe479310faafa4de8c7fe92e8cdf7ce5f
```

Comment:
while fin is not a submodule of miopen and it often gets out of sync, we still need to do this. Pretty soon this will go away, we are about to open source Fin.